test/widgets/message_list_test.dart

```diff
@@ -1758,6 +1759,79 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
+    group('User status', () {
+      void checkFindsStatusEmoji(WidgetTester tester, Finder emojiFinder) {
+        final statusEmojiFinder = find.ancestor(of: emojiFinder,
+          matching: find.byType(UserStatusEmoji));
+        check(statusEmojiFinder).findsOne();
+        check(tester.firstWidget<UserStatusEmoji>(statusEmojiFinder)
+          .neverAnimate).isTrue();
+
+        final senderRowFinder = find.ancestor(of: statusEmojiFinder,
+          matching: find.byType(SenderRow));
+        check(senderRowFinder).findsOne();
```

Comment:
nit: I think this would be just as clear if inlined, and a bit shorter. It's saying the status emoji has a SenderRow as an ancestor.

Reply:
Then also can drop the blank line above, and join this function's whole body in one stanza.